Assignment/solution_student/css/style.css

```diff
@@ -31,19 +31,252 @@ html {
 /* BASE STYLES
 /*******************************************/
 body {
-  color: black;
+  color: #121212;
   font-size: 16px;
-  background: white;
+  background: #ffffff;
+  margin: 0 auto;
+  font-family: 'Open Sans', sans-serif;
 }
-
 /******************************************
 /* LAYOUT
 /*******************************************/
+nav {
+  float: right;
+  margin-right: 50px;
+  margin-top: 20px;
+}
+
+nav li {
+  margin-right: 20px;
+  display: inline;
+}
+
+header {
+  width: 100%;
+  background-color: #033048;
+}
+
+.logo {
+  margin-top: 0;
+  dislay: inline-block;
+}
+
 
 /******************************************
-/* PAGES
+/* SECTIONS
+/*******************************************/
+
+nav a {
+  text-decoration: none;
+  color: #f9e42e;
+  font-style: 700;
+}
+
+.logo {
+  font-size: 24px;
+  font-weight: 800;
+}
+
+header .logo {
+  color: #f0efef;
+  display: inline-block;
+  margin-left: 50px;
+  margin-top: 10px;
+
+}
+
+.content-wrapper{  
+  background: #f0efef;
+  margin: 0 auto;
+  width: 75%;
+  padding: 25px 50px 20px;
+
+}
+
+.articles,
+.sidebar {
+  float: left;
+}
+
+.articles {
+  width: 75%;
+  padding-left: 50px;
+}
+
+.article_images {
+  width: 100%;
+}
+
+.sidebar {
+  width: 25%;
+  background: #f0efef;
+  padding-left: 20px;
+}
+
+.160x600 {
+  padding-left: 20px;
+  padding-top: 60px;
+  padding-right: 50px;
+  float: left;
+}
+
+ul {
+  margin: 0;
+  list-style-type: none;
+}
+
+
+/******************************************
+/* PAGES --- I've actually come to the conclusion
+that i have no idea what "pages" mean or what to potentially
+put here
 /*******************************************/
 
 /******************************************
 /* SECTIONS
-/*******************************************/
\ No newline at end of file
+/*******************************************/
+h2 {
+  color: #033048;
+  font-size: 40px;
+  font-weight: 600;
+  margin-bottom: 20px;
+}
+
+h3 {
+  margin: 0;
+  font-size: 25px;
+  /*32 per the design file looked crazy big*/
+  margin-top: 40px;
+  font-weight: 900;
+  color: #033048;
+}
+
+p {
+  color: #606161;
+  font-size: 24px;
+  line-height: 1.5;
+}
+
+.sidebar p {
+  font-size: 20px;
+  font-weight: 200;
+  color: #606161;
+  margin: 0;
+}
+
+.sidebar a {
+  font-size: 22px;
+  font-weight: 700;
+  text-decoration: none;
+  color: #033048;
+}
+
+article a {
+  color: #033048;
+  font-weight: 700;
+  font-size: 24px;
+  text-align: right;
+  /*I've text aligned right, but to 
```

Comment:
Ahh, okay, this might be a little confusing to explain, but here goes...
The text-align property will align the text within the element, not align the element itself, _so_, because you have an `<a>` just out in the open, and applied the text-align property to it, it's aligning the "read more" text to the right believe it or not, but because an `<a>` tag is an _inline_ element, it doesn't take up an entire row like a block-level element, it's only as wide as its own content, so you don't see a difference. What you'd probably want to do is wrap the `<a>` tag in a `p` or a `div`, and then give it a class to target (ie: `.read-more-link`) and apply `text-align:right;` to _it_ instead.